### DISCUSSION TITLE
How to configure relative path redirection?

### DISCUSSION THREAD

**Discussion by A81N9** [vuejs/router#2364] [Help and Questions] [answered]
I want `/menu` to redirect to `/menu /menu-1`
I must configure `redirect: '/menu/menu-1'`
How to configure relative path redirection?
Just linke this:

```js
[
  {
    path: '/',
    children: [
      {
        path: 'menu',
        // redirect: 'menu-1',
        children: [
          {
            path: 'menu-1',
          },
        ],
      },
    ],
  },
]
```

**Comment by posva** [comment, marked as answer]
Probably `redirect: to => to.path + '/' + 'menu-1'` or a similar approach that keeps query and hash

**Reply by A81N9** [reply]
Thank you for your reply.

But I think the official document example seems to be incorrect.

[redirect-and-alias](https://router.vuejs.org/guide/essentials/redirect-and-alias.html#Relative-redirecting)

![Screenshot_2024-09-22-10-27-28-596-edit_com.microsoft.emmx.jpg](https://github.com/user-attachments/assets/7823bd36-6df3-4506-a306-2a1817733665)

**Reply by posva** [reply]
Then the docs should be updated
